app/app.go

```diff
@@ -194,13 +193,8 @@ var (
 		mint.AppModuleBasic{},
 		distr.AppModuleBasic{},
 		gov.NewAppModuleBasic(append(
-			wasmclient.ProposalHandlers,
+			[]govclient.ProposalHandler{},
```

Comment:
Is ModuleBasics still needed?

> Previously, the ModuleBasics was a global variable that was used to register all modules' AppModuleBasic implementation. The global variable has been removed and the basic module manager can be now created from the module manager.

Reply:
I'll add that to the list of TODOs in the [epic](https://github.com/provenance-io/provenance/issues/1760).